client/src/components/productDetails.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import ReactDOM from "react-dom";
+class ProductDetails extends React.Component {
```

Comment:
This file reads like a stateless component (functional) but is written as stateful (class). Not something that needs to be changed now, but potentially a refactor in the future.